### PR TITLE
Bug #302 spinner label fix

### PR DESCRIPTION
**Pull request by wdo3650**
Fixed the label bug.

### CODE REVIEW

File: src/components/Spinner/Spinner.js [hunk 1]
```diff
@@ -52,8 +53,18 @@ fabric.Spinner = function(target) {
...
+            spinner = document.createElement("div");
+            spinner.className = "ms-Spinner";
+            _target.appendChild(spinner);
+        } else {
+            spinner = _target;
+        }
+
         offsetSize = eightSize;
         numCircles = 8;
+        parentSize = spinner.className.indexOf("large") > -1 ? 28 : 20;
```
Comment (mikewheaton) [inline src/components/Spinner/Spinner.js]: Do we think 28px and 20px is enough of a difference? Maybe we can find examples of this in product to see what sizes are most common, it just seems a little small to me.

Comment (mikewheaton) [inline src/components/Spinner/Spinner.js]: Never mind, these are the correct sizes from the design library.

**Comment by mikewheaton** [comment]
Looks good. One more reviewer and we can merge it in.

**Comment by Jahnp** [comment]
Looks good!

**Pull request merged**